src/containers/MyNdla/folderMutations.ts

```diff
@@ -257,6 +259,17 @@ const updateFolderStatusMutation = gql`
   }
 `;
 
+const copySharedFolderMutation = gql`
+  mutation copySharedFolder($folderId: String!, $destinationFolderId: String) {
+    copySharedFolder(
```

Comment:
Får en litt rar feilmelding her i editor, der det ikke ser ut som `copySharedFolder` eksisterer ... innspill?

Reply:
Kan hjelpe å restarte vscode for at den skal plukke opp alt av endringer i schema.